script/DeployRouter.s.sol

```diff
@@ -27,7 +31,7 @@ abstract contract DeployRouter is DeployBase {
         deployedAddress = cfg.deployedAddress;
         if (deployedAddress == UNDEPLOYED) {
             ICREATE3Factory factory = ICREATE3Factory(create3Factory);
-            bytes32 salt = keccak256('protocolink.router.v2');
+            bytes32 salt = keccak256('protocolink.router.v1');
```

Comment:
I thought the version number was an increment. Is there any reason why we decrease the version number for this deployment?

Reply:
As the create3 factory was redeployed using a new testing deployer, we can reset the salt.